upcloud_api/cloud_manager/server_mixin.py

```diff
@@ -152,23 +152,34 @@ def modify_server(self, uuid: str, **kwargs) -> Server:
             cloud_manager=self,
         )
 
-    def delete_server(self, uuid):
+    def delete_server(
+        self,
+        uuid: str,
+        delete_storages: bool = False,
+        backups: BackupDeletionPolicy = BackupDeletionPolicy.KEEP,
+    ):
         """
         DELETE '/server/UUID'. Permanently destroys the virtual machine.
 
-        DOES NOT remove the storage disks.
+        Does remove storage disks if delete_storages is defined as True.
+
+        Does remove backups of the attached storages if
```

Comment:
I think it would be better to describe these as actions that "user" takes

```
Remove storage disks by defining delete_storages true

Remove backups by defining BackupDeletionPolicy
```

Now that I think about it, might be better if the variable would be something like `BackupDeletionPlan`

Reply:
Unsure about the variable naming though. The plan doesn't really sound that great of a name

Reply:
Hmmm... Seems that other descriptions and docs are in this passive mode. So maybe we'll just keep it as is.

Reply:
Yeah, but we can re-visit them later if wanted.